### PR TITLE
Updating XCCDF mapper formatting

### PR DESCRIPTION
**Pull request by charleshu-8**
Signed-off-by: charleshu-mitre <chu@mitre.org>

### CODE REVIEW

**Comment by Rlin232** [comment]
> 

Not sure what to do about the SonarCloud complaining about the commented-out code. It is indeed useful to have in there for the sake of testing, so I'm of the opinion that we leave it in and ignore the SonarCloud stuff.

**Comment by Amndeep7** [comment]
> I'm of the opinion that we leave it in and ignore the SonarCloud stuff.

that's exactly what we're gonna do @Rlin232

File: libs/hdf-converters/src/xccdf-results-mapper.ts [hunk 1]
```diff
@@ -213,9 +218,7 @@ export class XCCDFResultsMapper extends BaseConverter {
       }
     },
     version: HeimdallToolsVersion,
-    statistics: {
-      duration: 0
-    },
+    statistics: {},
     profiles: [
       {
         name: {path: ['cdf:Benchmark.id', 'Benchmark.id']},
```
Comment (Amndeep7) [inline libs/hdf-converters/src/xccdf-results-mapper.ts:224]: @camdenmoors mentioned something about how `cdf:` is some sorta prefix that we can tell the parser to ignore.  talk to him to see what's up with that so we can ideally simplify this mapper.

**Review by Amndeep7 — CHANGES_REQUESTED** [review]
I would appreciate a chance to pair to see what the new output looks like just to confirm that it's all good.  Github don't wanna show diffs this large lol.

**Review by Amndeep7 — APPROVED** [review]

**Pull request merged**